packages/common/src/utils/test-utils.ts

```diff
@@ -65,4 +67,15 @@ export class TestUtils {
     ])
     return CID.create(1, SHA256_CODE, decodeMultiHash(body))
   }
+
+  static async anchorUpdate(ceramic: CeramicApi, stream: Stream): Promise<void> {
```

Comment:
```suggestion
  static async anchorUpdate(ceramic: Ceramic, stream: Stream): Promise<void> {
```

Reply:
This code actually won't work if it's given a `CeramicClient` it has to a be a Ceramic 'core' instance to be able to access the anchorService

Reply:
Better not create a dependency on Core, as it would create a circular dependency. Added a doc-comment with a warning.